### PR TITLE
Adds Babel Symptom as a stage 4 virus symptom. 

### PR DESCRIPTION
**Pull request by gbasood**
Causes subject to lose knowledge of all languages but one. Languages are restored on cure.

### CODE REVIEW

File: html/changelogs/Sood.yml [hunk 1]
```diff
@@ -1,2 +1,3 @@
 author: Sood
-changes: []
+changes:
+  rscadd: Adds a Babel Syndrome to the stage 4 virus symptoms. Causes subject to lose all knowledge of all languages but one.
```
Comment (kol1th) [inline html/changelogs/Sood.yml]: :joy:

Comment (gbasood) [inline html/changelogs/Sood.yml]: ?????

**Comment by Exxion** [comment]
Sood
Are
Are you OK?

**Comment by gbasood** [comment]
this is what happens when you:
- Use oldcode on accident
- Copypaste to newcode and fuck it up several times
- Use travis to test your code

**Comment by ghost** [comment]
https://github.com/d3athrow/vgstation13/pull/6262/files or https://github.com/d3athrow/vgstation13/blob/Bleeding-Edge/code/modules/virus2/effect.dm might help you out some here

**Comment by gbasood** [comment]
It's not that I don't get the code I just fucked something up and I'm being lazy and copypasting instead of rewriting
I'll fix it another day too tired

**Comment by ComicIronic** [comment]
But the destruction of the Tower of Babel led to an increase in the number of languages.

**Comment by The-Albinobigfoot** [comment]
Shouldn't it give you a random language to speak exclusively?

**Comment by gbasood** [comment]
@ComicIronic reference != 100% accurate
@The-Albinobigfoot congratulations on your RNG roll ou now speak only changeling in a ling round

**Comment by NigglyWiggly** [comment]
Changeling isn't a language you goober
Giving you a random language would be great though, just make it choose from a list of languages understood by silicons

**Comment by gbasood** [comment]
there is actually a ling language datum

**Comment by clusterfack** [comment]
It uses handle inherent languages instead

**Comment by clusterfack** [comment]
Wee woo wee woo

**Comment by gbasood** [comment]
@clusterfack what did he mean by this

**Comment by clusterfack** [comment]
It's the compile police

**Pull request merged**